src/pages/display/ClubsPage.jsx

```diff
@@ -37,15 +89,66 @@ const ClubsPage = () => {
               </p>
             </div>
           </div>
+          <div
+            style={{
+              display: "flex",
+              justifyContent: "center",
+              alignItems: "center",
+            }}
+          >
+            <button
```

Comment:
If possible also remove these inline styles and add this to a separate `.css` file.